src/index.ts

```diff
@@ -13,6 +14,27 @@ const S = [0n, 1n, 2n, 4n, 8n, 16n];
 type Quadbin = bigint;
 type Tile = {x: number; y: number; z: number};
 
+function tileToLongitude(tile: ReturnType<typeof cellToTile>, offset: number) {
+  const {x, z} = tile;
+  return 180 * ((2.0 * (x + offset)) / (1 << z) - 1.0);
+}
+
+function tileToLatitude(tile: ReturnType<typeof cellToTile>, offset: number) {
+  const {y, z} = tile;
+  const expy = Math.exp(-((2.0 * (y + offset)) / (1 << z) - 1) * Math.PI);
+  return 360 * (Math.atan(expy) / Math.PI - 0.25);
+}
+
+function cellToBoundingBox(cell: bigint) {
```

Comment:
Still needs rename to `cellToBoundary` (including docs)

Reply:
This is not the function we export, but the one below 👍